doc/source/security/data-scrubbing.rst

```diff
@@ -0,0 +1,54 @@
+.. _data-at-rest-encryption.data-scrubbing:
+.. _data-scrubbing:
+
+Data Scrubbing
+================================================================================
+
+:Availability: This feature is **Experimental** quality
+
+While data encryption ensures that the existing data are not stored in plain
+form, the data scrubbing literally removes the data once the user decides they
+should be deleted. Compare this behavior with how the ``DELETE`` statement works
+which only marks the affected data as *deleted* - the space claimed by this data
+is overwritten with new data later.
+
+Once enabled, data scrubbing works automatically on each tablespace
+separately. To enable data scrubbing, you need to set the following variables:
+
+- :variable:`innodb-background-scrub-data-uncompressed`
+- :variable:`innodb-background-scrub-data-compressed`
```

Comment:
Didn't we drop the entire encryption threads stuff for 5.7? Because background data scrubbing depends on that feature.

Reply:
Dropped.